src/head/Columns.js

```diff
@@ -25,19 +23,20 @@ function getColumns(columnsDefinitions) {
     } = columnsDefinitions;
 
     return columns && columns.length ? columns.map(column => {
-
-            column.sorterComponent = sorterComponent;
-            column.onSort = onSort;
-
-            return (
-                <Column
-                    className={columnClassName}
-                    key={sid.generate()}
-                    column={column}
-                />
-            );
-
-        }) : null;
+        const newColumnDefinition = Object.assign({}, column, {
+            sorterComponent,
+            onSort,
+            isSortable: column.isSortable === undefined ? true : column.isSortable
```

Comment:
I think that Column component, not Columns, should determine, if it is sortable or not. Parent should not care about props computation of children. It is not responsibility of the Columns component.